components/promotion/base/namespace.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: rhtap-staging-testing
```

Comment:
I would expect a placeholder here, that's kustomized for a specifc staging instance.
That way it will be easy to implement production when needed.

In the same way, I would expect placeholders for each namespace references (`namespace: rhtap-staging-testing`).

However, I'm fine with merging as is and refactoring when needed.